### DISCUSSION TITLE
resetXform command

### DISCUSSION THREAD

**Discussion by kimuss** [Autodesk/maya-usd#3543] [General]
Hello there, 
I was wondering if there is a "simple" way to reset(freeze?)  xform on an asset, like a dropdwon menu or similar
so if I have an asset with this xform:
![immagine](https://github.com/Autodesk/maya-usd/assets/54888290/47ec80c0-b6e5-4dbe-8f6d-46d1b7a7e752)
can I reset this trasform to freeze the traslate and have new position with 0,0,0?
thank you for your time

**Comment by wallworm** [comment]
Thank you for sharing the request. We will consider this.
